### DISCUSSION TITLE
Wireless controller always auto selects

### DISCUSSION THREAD

**Discussion by sajmonides** [Belphemur/SoundSwitch#1012] [FAQ]
I have only 2 devices selected - headphones and speakers. 
Whenever I plug in a controller, no matter what, it always gets automatically added to the selected section and I need to either manually de-select it or loop through one extra item with my shortcut to switch between them.
Even after de-selecting, next time I plug in the same controller to the same USB port, it is automatically re-added to the selected part.
I have the "Auto select new devices" option unchecked in settings.
Is there a way to prevent this?

**Comment by Belphemur** [comment]
Hello,

Could you attach the logs and config of the application when it happens ?

* logs:  `%appdata%/SoundSwitch/logs`
* config: `%appdata%/SoundSwitch/SoundSwitchConfiguration.json`

This isn't supposed to happen when the setting is disabled.
